dotnet/src/Skills/Skills.Planning/SequentialPlanner/PlannerConfig.cs

```diff
@@ -2,7 +2,7 @@
 
 using System.Collections.Generic;
 
-namespace Microsoft.SemanticKernel.Planning.Planners;
+namespace Microsoft.SemanticKernel.Skills.Planning.SequentialPlanner;
 
 /// <summary>
 /// Common configuration for planner instances.
```

Comment:
I think this or parts of it ought to be common.

Reply:
the concept of included/excluded functions, relevance and maxrelevance is a feature of the sequential planner, I don't think we should assume that every planner will use it. The basic planner for example doesn't use any of them